src/shared/download-file.ts

```diff
@@ -0,0 +1,49 @@
+function downloadFile(
+  content: string,
+  type: string,
+  filename: string,
+  document: Document,
+): void {
+  const blob = new Blob([content], { type });
+  const url = URL.createObjectURL(blob);
```

Comment:
After the download has been performed, I believe we're supposed to use `revokeObjectURL` to release the blob.